packages/@okta/i18n/src/properties/login.properties

```diff
@@ -929,10 +929,11 @@ oie.email.verify.primaryButton = Send me an email
 oie.email.verify.subtitleWithEmailAddress = Verify with an email link or enter a code sent to <$1>{0}</$1>.
 oie.email.verify.subtitleWithoutEmailAddress = Verify with an email link or enter a code sent to your email.
 # {0} is a user's email address
-oie.email.verify.alternate.magicLinkToEmailAddress = An email magic link was sent to <$1>{0}</$1>. 
+oie.email.verify.alternate.magicLinkToEmailAddress = We sent an email to <$1>{0}</$1>. 
```

Comment:
Why are we changing the existing UI strings?

Reply:
@lokeshhemrajani-okta are we not going to show `An email magic link was sent` in any other case now? are we scrapping it ?

Reply:
@IanHakes-okta can you please reply on this? I updated as per comment in the story OKTA-454634

Reply:
Change based on 454634 - @bhagyaprabhakar-okta asked for text to resolve the scenario where "the customer is able to customize the email template and not have the email link at all.  In this case, the authenticator screen should not include any instructions to click link in email."

Reply:
@IanHakes-okta @bhagyaprabhakar-okta question is are we scrapping `An email magic link was sent` text from all use cases ?

@lokeshhemrajani-okta must verify this in okta-core before merging to widget.. there might selenium test failures for this text change.

Reply:
Yes.  The intention is to use "active voice" and have a more direct instruction to the user.